File: Src/java/cql-to-elm/src/jvmTest/java/org/cqframework/cql/cql2elm/fhir/v18/BaseTest.java
package org.cqframework.cql.cql2elm.fhir.v18;

import java.io.IOException;
import org.cqframework.cql.cql2elm.TestUtils;
import org.junit.jupiter.api.Test;

class BaseTest {
    @Test
    void fhirHelpers() throws IOException {
        TestUtils.runSemanticTest("fhir/v18/TestFHIRHelpers.cql", 0);
    }

    @Test
    void fhir() throws IOException {
        TestUtils.runSemanticTest("fhir/v18/TestFHIR.cql", 0);
    }

    @Test
    void fhirWithHelpers() throws IOException {
        TestUtils.runSemanticTest("fhir/v18/TestFHIRWithHelpers.cql", 0);
    }
}

File: Src/java/cql-to-elm/src/jvmTest/kotlin/org/cqframework/cql/cql2elm/fhir/v18/BaseTest.kt
package org.cqframework.cql.cql2elm.fhir.v18

import java.io.IOException
import org.cqframework.cql.cql2elm.TestUtils.runSemanticTest
import org.junit.jupiter.api.Test

internal class BaseTest {
    @Test
    @Throws(IOException::class)
    fun fhirHelpers() {
        runSemanticTest("fhir/v18/TestFHIRHelpers.cql", 0)
    }

    @Test
    @Throws(IOException::class)
    fun fhir() {
        runSemanticTest("fhir/v18/TestFHIR.cql", 0)
    }

    @Test
    @Throws(IOException::class)
    fun fhirWithHelpers() {
        runSemanticTest("fhir/v18/TestFHIRWithHelpers.cql", 0)
    }
}
